lib/authlogic/acts_as_authentic/password.rb

```diff
@@ -109,13 +109,22 @@ def check_passwords_against_database(value = nil)
         # transition to a better crypto provider without causing your users any
         # pain.
         #
-        # * <tt>Default:</tt> CryptoProviders::SCrypt
+        # * <tt>Default:</tt> There is no longer a default value. Prior to
+        #   Authlogic 6, the default was `CryptoProviders::SCrypt`. If you try
+        #   to read this config option before setting it, it will raise a
+        #   `NilCryptoProvider` error. See that error's message for further
+        #   details, and rationale for this change.
```

Comment:
Should we raise an error on boot instead of lazily? Seems more certain that people will see it.